workspaces/client/app/routes/dashboard/admin/equipments/internal/hooks/use-create-equipments.tsx

```diff
@@ -0,0 +1,40 @@
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+import { useRepository } from "~/hooks/use-repository";
+import { useToast } from "~/hooks/use-toast";
+import type { EquipmentWithOwner } from "~/types/equipment";
+
+export const useCreateEquipment = () => {
+	const { equipmentRepository } = useRepository();
+	const queryClient = useQueryClient();
+	const { pushToast } = useToast();
+	return useMutation({
+		mutationFn: async (
+			equipment: Omit<
+				EquipmentWithOwner,
+				"id" | "createdAt" | "updatedAt" | "owner"
+			>,
+		) => {
+			await equipmentRepository.createEquipment({
+				name: equipment.name,
+				description: equipment.description ?? undefined,
+				ownerId: equipment.ownerId,
+			});
+			return equipment;
+		},
+		onSuccess: (equipment) => {
```

Comment:
```suggestion
		onSuccess: (_, equipment) => {
```
これでいけると思っている (ref: admin/certifications/interna/hooks/use-review-request.ts)